From 44f4411474d26e3f8fc1f468a867111fbc4361f5 Mon Sep 17 00:00:00 2001
From: Brian Bennett <brian.bennett@joyent.com>
Date: Thu, 16 Nov 2017 17:53:50 -0800
Subject: [PATCH] joyent/manta-thoth#157 building in manta is broken Reviewed
 by: Bryan Cantrill <bryan@joyent.com> Approved by: Bryan Cantrill
 <bryan@joyent.com>

---
 tools/build-release.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/build-release.sh b/tools/build-release.sh
index 2078e51..7549057 100755
--- a/tools/build-release.sh
+++ b/tools/build-release.sh
@@ -35,7 +35,7 @@ if [[ $uname == sunos ]]; then
 fi
 (
     cd opt/custom/thoth
-    npm install smartdc thoth
+    npm install smartdc manta-thoth
 )
 
 tar zcf "$tar" opt
-- 
2.21.0

